Project.toml

```diff
@@ -1,17 +1,18 @@
 name = "Intervals"
 uuid = "d8418881-c3e1-53bb-8760-2df7ec849ed5"
+license = "MIT"
+desc = "Non-iterable ranges"
 authors = ["Curtis Vogt <curtis.vogt@gmail.com>", "Gem Newman <spurll@gmail.com>"]
```

Comment:
We should also update this to be "Invenia Technical Computing"

Reply:
I'm still not sure how I feel about this. Listing the authors is helpful for figuring out who are the major contributors. I guess you could just use the contributors graph to get that info.